Driverless AI/ES/introducción-a-machine-learning-automático-con-driverless-ai/introducción-a-machine-learning-automático-con-driverless-ai.md

```diff
@@ -455,37 +457,37 @@ Si desea realizar un recorrido rápido por la página **Experiments** (Experimen
 10. [Weight column](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#weight-column) (Columna de peso) - Columna que indica el peso de observación (también conocido como peso de muestra o fila), si corresponde.
 11. [Time Column](http://docs.h2o.ai/driverless-ai/1-8-lts/docs/userguide/experiment-settings.html?highlight=accuracy#time-column) (Columna de tiempo) (DESACTIVADO de forma predeterminada): proporciona un orden de tiempo (marcas de tiempo para las observaciones). Se usa cuando los datos tienen una alta dependencia del tiempo (como la estacionalidad o tendencia), y desea tratar este problema como un problema de serie temporal.
-
+ 
 Continuando con nuestro experimento:
-
+ 
 Nombre su experimento de la siguiente manera: Tutorial de Clasificación Titanic
-
+ 
 5\. Haga clic en **Dropped Columns**, retire las siguientes columnas: Passenger_Id (Id. De pasajero), name_with_salutations (nombre_con_saludos), name_without_salutations (nombre_sin_saludos), boat (bote), body (cuerpo) y home.dest (destino de origen). Luego seleccione **Done** (Listo).
-
+ 
 ![train-set-drop-columns](assets/train-set-drop-columns.jpg)
-
+ 
 Estos atributos (columnas) se eliminaron para crear un conjunto de datos más limpio. Los atributos como el bote y el cuerpo están excluidos porque son indicadores claros de que un pasajero sobrevivió y pueden conducir a la fuga de datos. Para nuestro experimento, la columna de sobrevivencia será suficiente para crear un modelo.
-
+ 
 Un conjunto de datos limpios es esencial para la creación de un buen modelo de predicción. El proceso de limpieza de datos debe hacerse con todos los conjuntos de datos para eliminar cualquier conjunto de observaciones no deseadas, errores estructurales, valores atípicos no deseados o datos faltantes.
-
+ 
 6\. Seleccione **Test Dataset** y luego haga clic en **titanic_test**
-
+ 
 ![add-test-set](assets/add-test-set.jpg)
-
-7\. Ahora seleccione la **Target Column**. En nuestro caso, la columna será 'survived' (sobrevivió).
-
+ 
+7\. Ahora seleccione **Target Column**. En nuestro caso, la columna será 'survived' (sobrevivió).
+ 
 ![train-set-drop-name-column](assets/train-set-drop-name-column.jpg)
-
-El atributo "sobrevivió" fue seleccionado porque, como compañía de seguros, queremos saber qué otros atributos pueden contribuir a la supervivencia de los pasajeros a bordo de un barco e incorporarlo a nuestras tarifas de seguro.
-
+ 
+El atributo "sobrevivió" fue seleccionado porque, como empresa de seguros, queremos saber qué otros atributos pueden contribuir a la supervivencia de los pasajeros a bordo de un barco e incorporarlo a nuestras tarifas de seguro.
```

Comment:
You called this attribute "sobrevivido" earlier on. I think "sobrevivió" sounds better.